test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -351,19 +347,11 @@ UCS_TEST_SKIP_COND_P(test_ucp_perf, envelope, has_transport("self"))
     /* coverity[tainted_string_argument] */
     ucs::scoped_setenv atomic_mode("UCX_ATOMIC_MODE", atomic_mode_str);
 
-    if (ucs_arch_get_cpu_model() == UCS_CPU_MODEL_ARM_AARCH64) {
-        test.max *= UCT_ARM_PERF_TEST_MULTIPLIER;
-        test.min /= UCT_ARM_PERF_TEST_MULTIPLIER;
-    } else {
-        test.max *= UCT_PERF_TEST_MULTIPLIER;
-        test.min /= UCT_PERF_TEST_MULTIPLIER;
-    }
-    test.iters = ucs_min(test.iters, max_iter);
-
+    test.iters         = ucs_min(test.iters, max_iter);
     test.send_mem_type = UCS_MEMORY_TYPE_HOST;
     test.recv_mem_type = UCS_MEMORY_TYPE_HOST;
 
-    run_test(test, 0, check_perf, "", "");
+    run_test(test, 0, false, "", "");
```

Comment:
can we remove `check_perf` from `run_test` now?

Reply:
It is possible, if we disable checking performance in other tests.